__tests__/api/api.test.js

```diff
@@ -230,14 +230,85 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 });
 
-describe("ALL /notapath", () => {
-  test("404: responds with a status of 404 and a custom message when the path is not found", () => {
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: responds with a status of 201 and the comment object that has been sent", () => {
+    const newComment = {
+      body: "The answer is doughnuts",
+      username: "butter_bridge"
+    };
     return request(app)
-      .get("/api/banana")
-      .expect(404)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        const { comment } = response.body;
+        expect(comment).toEqual({
```

Comment:
a better matcher here might be toMatchObject if you are only going to check a some of the keys, it might be worth checking it has a created at and votes too, particularly because you know the votes will default to 0